tests/restore/dataset/test_dataset_restore_items_generator.py

```diff
@@ -111,24 +111,6 @@ def test_should_not_create_restore_items_for_deleted_backups(self):
         # then
         self.assertEqual(generated_restore_items, [[restore_item]])
 
-    def test_should_create_restore_item_for_partition(
```

Comment:
I would left this tests, but edit what it expects. 

We still use partitionId in source table, only we don't specified them for target.  (it is how it was done in BackupCreator class - I believe we should do the same for restore also).